src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -262,14 +262,23 @@ public void updateFilteredModuleList(Predicate<Module> predicate) {
 
     @Override
     public ObservableList<Schedule> getFilteredScheduleList() {
+
         return filteredSchedule;
     }
 
+    @Override
+    public ObservableList<Schedule> getAllScheduleList() {
+        return new FilteredList<>(this.addressBook.getScheduleList());
+
+    }
+
 
     @Override
     public void updateFilteredScheduleList(Predicate<Schedule> predicate) {
+        // filteredSchedule = new FilteredList<>(this.addressBook.getScheduleList());
         requireNonNull(predicate);
-        filteredSchedule = new FilteredList<>(addressBook.getScheduleList());
```

Comment:
If we delete this part, will adding, editing, or deleting a schedule still show the updated schedule list?

Reply:
Yup. Actually I change it to be "filteredSchedule.setPredicate(predicate)"